test/state/host.cpp

```diff
@@ -268,14 +275,6 @@ evmc::Result Host::create(const evmc_message& msg) noexcept
 
     new_acc->just_created = true;
 
-    // Clear the new account storage, but keep the access status (from tx access list).
-    // This is only needed for tests and cannot happen in real networks.
-    for (auto& [k, v] : new_acc->storage) [[unlikely]]
```

Comment:
Because the storage is empty now we don't have to do anything with it. This is really nice because this eliminates the only case for storage traversal in EVM.

Reply:
Not easy check because of the access list... so I'll pass.